py_ecc/fields/field_elements.py

```diff
@@ -355,7 +366,7 @@ class FQ12(FQP):
     The 12th-degree extension field
     """
     degree = 12
-    FQ12_MODULUS_COEFFS = None
+    FQ12_MODULUS_COEFFS = None  # type: Tuple[int, ...]
```

Comment:
It *appears* that these are fixed length?  If that is correct can you update this to be `Tuple[int, int, int, ... <however many there are>]` so it's exact.